app/src/main/java/com/cmput301w23t40/capturetheqr/MapActivity.java

```diff
@@ -19,9 +38,47 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_map);
 
+        /* Copied from
+         * https://developers.google.com/maps/documentation/android-sdk/map
+         */
+        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
+                .findFragmentById(R.id.frgmt_qrMap);
+        mapFragment.getMapAsync(this);
+
+        playerLocation = new PlayerLocation();
+
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
+    /**
+     * Implement onMapReady callback for OnMapReadyCallback
+     * and initialize the map UI
+     * @param googleMap
+     */
+    @Override
+    public void onMapReady(GoogleMap googleMap) {
+        CameraUpdate camPosition = CameraUpdateFactory.newLatLngZoom(playerLocation.getLocation(), 18);
```

Comment:
Please add a few more comments to this method. ie what is the purpose of `18` in `CameraUpdate camPosition = CameraUpdateFactory.newLatLngZoom(playerLocation.getLocation(), 18);` 

This may just help with readability and debugging for the rest of the team later

Reply:
Done